README.md

```diff
@@ -1564,20 +1622,15 @@ await client.checkout.orderNext({ orderToken })
 // Step three - pick a payment method
 const payment = (await client.checkout.paymentMethods({ orderToken })).success()
 
-await client.checkout.orderUpdate({ orderToken }, {
-  order: {
-    payments_attributes: [{
-      payment_method_id: payment.data[0].id
-    }]
-  },
-  payment_source: {
-    [payment.data[0].id]: {
-      number: '4111111111111111',
-      month: '1',
-      year: '2022',
-      verification_value: '123',
-      name: 'John Doe'
-    }
+await client.checkout.addPayment({ orderToken }, {
+  payment_method_id: payment.data[0].id,
+  source_attributes: {
+    gateway_payment_profile_id: payment.data[0].source.gateway_payment_profile_id,
+    cc_type: payment.data[0].source.cc_type,
+    last_digits: payment.data[0].source.last_digits,
+    month: payment.data[0].source.month,
+    year: payment.data[0].source.year,
+    name: payment.data[0].source.name
```

Comment:
@aplegatt this data is not coming from payment method - this is a payment, a completely different entity, this needs to be set by hand

Reply:
@damianlegawiec How should I present it in the readme file then?

Reply:
@aplegatt use the example data - https://github.com/spree/spree-storefront-api-v2-js-sdk/pull/273/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1326-R1331